src/libcore/benches/lib.rs

```diff
@@ -20,6 +20,6 @@ extern crate test;
 mod any;
 mod hash;
 mod iter;
-mod mem;
```

Comment:
Why is this removed?

Reply:
It doesn't exist.

Reply:
@alexcrichton How was this able to get past CI? Do we not run benches as part of libcore's tests?

Reply:
I don't think we run benches in CI.  The file was moved in #44943.